.github/workflows/deploy.yml

```diff
@@ -283,6 +279,21 @@ jobs:
           AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
           AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
           GCP_GEOLOCATION_API_KEY: ${{ secrets.GCP_GEOLOCATION_API_KEY_PRODUCTION }}
+          S3_BUCKET: ${{ vars.STAGING_S3_BUCKET }}
+
+      - name: Build production desktop app
```

Comment:
Why'd you split these steps up?

Reply:
The S3 bucket name is different between the two. I agree it seems to be a matter of preference as I prefer the separate steps for readability.

I can re-dry it back up.